front/src/reducers/index.ts

```diff
@@ -98,11 +114,11 @@ export interface RootState {
 export const rootInitialState: RootState = {
   user: userInitialState,
   map: mapInitialState,
-  [editorSlice.name]: editorInitialState,
+  mapViewer: mapViewerInitialState,
+  [editorSlice.name]: { ...editorInitialState, ...infraState },
```

Comment:
No need to use infraState since it is already added in editorInitialState
```suggestion
  [editorSlice.name]: { ...editorInitialState },
```

Reply:
Done